src/OptionHandler.cpp

```diff
@@ -114,16 +114,65 @@ static std::unique_ptr<AudioFileReader> createAudioFileReader(
 
 //------------------------------------------------------------------------------
 
-static std::unique_ptr<ScaleFactor> createScaleFactor(const Options& options)
+// Returns the frame count of a given audio. Frame is block of samples, one for each channel.
+
+static std::pair<bool, long> getFrameCount(
```

Comment:
We already have a `getDuration` function that does the same job, only returning duration in seconds, rather than frames. I'd prefer not to duplicate code where possible.

Reply:
So what you suggest? 
1. Remove getFrameCount
2. Replace usage of "FrameCount" with duration*sample_rate?

Reply:
I was thinking to modify `getDuration` to return frames instead of seconds, then divide by sample rate where we need seconds